apps/client/src/components/ContactSentence.tsx

```diff
@@ -1,15 +1,33 @@
-import React from "react";
+import { Paragraph } from "@datapunt/asc-ui";
+import React, { ReactElement } from "react";
 
 import PhoneNumber from "./PhoneNumber";
 
 type Props = {
-  link?: boolean;
   eventName?: string;
+  inline?: boolean;
+  link?: boolean;
+  openingSentence?: string;
 };
 
-export default ({ link = true, eventName }: Props) => (
-  <>
-    Bel dan de gemeente op <PhoneNumber {...{ link, eventName }} />, maandag tot
-    en met vrijdag van 08.00 uur tot 18.00 uur.
-  </>
+type WithParagraphProps = {
+  children: ReactElement;
+  inline: boolean;
+};
+
+const WithParagraph = ({ children, inline }: WithParagraphProps) =>
+  !inline ? <Paragraph>{children}</Paragraph> : children;
+
+export default ({
+  eventName,
+  inline = false,
+  link = true,
```

Comment:
This is not so nice. `link` is optional, but default on `true`. Now you need to unset it by using `link={false}`. Not your fault, it's already in chappie2.0. I've added this to the refactor list.